README.md

```diff
@@ -89,17 +89,255 @@ Name of the application pool.
+
+Note: Because process accounting in IIS uses Windows job objects to monitor CPU times for the whole process, process accounting will only log and throttle applications that are isolated in a separate process from IIS
+
+#### `cpu_smp_affinitized`
+
+Specifies whether a particular worker process assigned to an application pool should also be assigned to a given CPU. This property is used together with the `cpu_smp_processor_affinity_mask` and `cpu_smp_processor_affinity_mask2` properties. Valid options `true` or `false`
+
+#### `cpu_smp_processor_affinity_mask`
+
+Specifies the hexadecimal processor mask for multi-processor computers, which indicates to which CPU the worker processes in an application pool should be bound. Before this property takes effect, the `cpu_smp_affinitized` property must be set to true for the application pool.
+
+Note: On 64-bit computers, the cpu_smp_processor_affinity_mask property contains the low-order DWORD for the processor mask, and the `cpu_smp_processor_affinity_mask2` property contains the high-order DWORD for the processor mask. On 32-bit computers, the `cpu_smp_processor_affinity_mask2` property has no effect.
+
+If you set the value to 1 (which corresponds to 00000000000000001 in binary), the worker processes in an application pool run on only the first processor. If you set the value to 2 (which corresponds to 0000000000000010 in binary), the worker processes run on only the second processor. If you set the value to 3 (which corresponds to 0000000000000011 in binary) the worker processes run on both the first and second processors.
+
+Note: Do not set this property to 0. Doing so disables symmetric multiprocessing (SMP) affinity and creates an error condition. This means that processes running on one CPU will not remain affiliated with that CPU throughout their lifetime
+
+#### `cpu_smp_processor_affinity_mask2`
+Specifies the high-order DWORD hexadecimal processor mask for 64-bit multi-processor computers, which indicates to which CPU the worker processes in an application pool should be bound. Before this property takes effect, the `cpu_smp_affinitized` property must be set to true for the application pool.
+
+Note: On 64-bit computers, the cpu_smp_processor_affinity_mask property contains the low-order DWORD for the processor mask, and the `cpu_smp_processor_affinity_mask2` property contains the high-order DWORD for the processor mask. On 32-bit computers, the `cpu_smp_processor_affinity_mask2` property has no effect
+
+#### `identity_type`
+
+Specifies the account identity under which the application pool runs. Note: Starting in IIS 7.5 the default value is `:ApplicationPoolIdentity`. (In IIS 7.0 the default value was `:NetworkService`).  Valid options `:ApplicationPoolIdentity`, `:LocalService`, `:LocalSystem`, `:NetworkService`, or `:SpecificUser`
```

Comment:
These values are likely passed as strings, so should be in single quotes rather than backticks.